addons/point_of_sale/views/report_saledetails.xml

```diff
@@ -60,7 +60,7 @@
             <br/>
 
             <h3>Taxes</h3>
-            <table  class="table table-sm">
+            <table  class="table table-sm" id="taxesTable">
```

Comment:
we cannot do that in stable as it won't be modified in the existing db, you should remove this change and do an xpath (maybe ugly) in the belgian module